### PR TITLE
Developer virtualenv bootstrap scripts.

### PR DESCRIPTION
**Pull request by kuba**

### CODE REVIEW

**Comment by kuba** [comment]
I encourage all active developers on the team to test out this script and please report back your experience:

```bash
./bootstrap/dev/venv.sh  # should backup your current venv and set up fresh new one
source venv/bin/activate
# do some development
```

@pde, @jdkasten, @bmw, @schoen?

**Comment by bmw** [comment]
Since I'm developing on Arch where Python 3 is the default, my `virtualenv` script is named `virtualenv2`. With that one exception, the script worked great for me.

**Comment by kuba** [comment]
As discussed on IRC, fixed the Arch issue by requiring `python-virtualenv` in the Arch bootstrap script.

File: docs/contributing.rst [hunk 1]
```diff
@@ -7,38 +7,39 @@ Contributing
...
-This step should prepend you prompt with ``(venv)`` and save you from
-typing ``./venv/bin/...``. It is also required to run some of the
-`testing`_ tools. Virtualenv can be disabled at any time by typing
-``deactivate``. More information can be found in `virtualenv
-documentation`_.
+.. code-block:: shell
 
-Install the development packages:
+   ./bootstrap/dev/venv.sh
+   ./bootstrap/dev/venv3.sh
```
Comment (bmw) [inline docs/contributing.rst]: As we are not currently developing and testing for Python 3+, I don't think this should be here.

File: docs/contributing.rst [hunk 2]
```diff
@@ -7,38 +7,39 @@ Contributing
...
 
-.. code-block:: shell
+Both of the commands suggest to activate the virtualenv (you can
+activate one at a time only):
```
Comment (bmw) [inline docs/contributing.rst]: If `venv3.sh` line above is removed, this can be removed as well.

File: docs/contributing.rst (hunk 1 above)
Comment (kuba) [inline docs/contributing.rst]: OK, fixed.

File: docs/contributing.rst (hunk 2 above)
Comment (kuba) [inline docs/contributing.rst]: ditto, fixed

**Comment by bmw** [comment]
LGTM. Perhaps something like this should be created for regular users as well.

**Comment by kuba** [comment]
Working on script for regular users. It will get the latest release from PyPI.

**Pull request merged**